helm/prometheus-rules/templates/alerting-rules/kyverno.all.rules.yml

```diff
@@ -0,0 +1,76 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  creationTimestamp: null
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+  name: kyverno.all.rules
+  namespace: {{ .Values.namespace  }}
+spec:
+  groups:
+  - name: webhooks
+    rules:
+    - alert: KyvernoWebhookHasNoAvailableReplicas
+      annotations:
+        description: 'Kyverno has no available replicas but webhooks are present.'
+        opsrecipe: kyverno-webhooks/
+      expr: sum(kube_validatingwebhookconfiguration_info{validatingwebhookconfiguration=~"kyverno-.*"}) > 0 and sum(kube_deployment_status_replicas{deployment=~"kyverno|kyverno-admission-controller"}) == 0
+      for: 5m
```

Comment:
On a second look, 5m might be too noisy for an out of business hours page. We can try this and adjust if it starts waking folks up for silly reasons

Reply:
Ooooooh you are right

Reply:
Well, but also this is something that could break the cluster. Maybe we could set it to 15/30m?